packages/utilities/psammead-assets/index.test.jsx

```diff
@@ -41,3 +51,50 @@ describe('Psammead assets', () => {
     );
   });
 });
+
+const { news, yoruba, igbo, pidgin, thai } = svgs;
```

Comment:
Everything below line 55 can probably be moved to `packages/utilities/psammead-assets/src/svgs.test.jsx`